python/lsst/sims/GalSimInterface/galSimCelestialObject.py

```diff
@@ -120,9 +142,32 @@ def galSimType(self, value):
         raise RuntimeError("You should not be setting galSimType on the fly; " \
                            + "just instantiate a new GalSimCelestialObject")
 
+    @property
+    def npoints(self):
+        return self._float_values[15]
```

Comment:
npoints is an int.  Probably should either (a) leave that out of `_float_values` and keep it as a regular attribute or (b) explicitly cast back to an int here in the getter.  

Double precision should be fine to capture any int value we would use without weird rounding errors, so (b) is probably fine.  But it *might* cause problems if it's allowed to turn into a float.

Reply:
It looks like Francois already casts it into an `int` when it gets used

https://github.com/lsst/sims_GalSimInterface/blob/master/python/lsst/sims/GalSimInterface/galSimInterpreter.py#L465

I'm not sure how zealous we want to be in cutting every possible bit of python overhead from our memory footprint.  If you still think it would be safer to go back to storing `self._npoints` as an `int`, I can do that.

Reply:
It might be slightly paranoid of me, but if you want to be absolutely sure that we're safe, I'd make the getter here return `int(self._float_values[15] + 0.5)`.  Just to make sure some floating point imprecision doesn't convert an int into something like `3.3999999999987e1` and get messed up by the (implicit) floor in the conversion back to int.

Reply:
But again, I do think that we would be safe even without that.  I'm just not 100% positive.  But rounding will definitely be safe over the range of npoints values we'll ever be using.

Reply:
I have implemented your suggestion.